stable/external-dns/README.md

```diff
@@ -26,21 +26,21 @@ The following tables lists the configurable parameters of the external-dns chart
 | `aws.access_key`          | `AWS_ACCESS_KEY_ID` to set in the environment (optional).                                                                  | `""`                                               |
 | `aws.secret_key`          | `AWS_SECRET_ACCESS_KEY` to set in the environment (optional).                                                              | `""`                                               |
 | `aws.region`              | `AWS_DEFAULT_REGION` to set in the environment (optional).                                                                 | `us-east-1`                                        |
-| `cloudflare.api_key`      | `CF_API_KEY` to set in the environment (optional).                                                                 | `""`                                        |
-| `cloudflare.email`        | `CF_API_EMAIL` to set in the environment (optional).                                                                 | `""`                                        |
+| `cloudflare.api_key`      | `CF_API_KEY` to set in the environment (optional).                                                                         | `""`                                               |
```

Comment:
Remove EOL tabs

Reply:
What tabs? The file was formatted with spaces to look good in editor, except for those two line. So I made them match the rest of formatting.

Reply:
Yep just spaces